internal/api/grpc/policy/v2/policy.go

```diff
@@ -0,0 +1,305 @@
+package policy
+
+import (
+	"context"
+
+	"google.golang.org/protobuf/types/known/durationpb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/zitadel/zitadel/internal/api/authz"
+	"github.com/zitadel/zitadel/internal/api/grpc/text"
+	"github.com/zitadel/zitadel/internal/domain"
+	"github.com/zitadel/zitadel/internal/query"
+	object "github.com/zitadel/zitadel/pkg/grpc/object/v2alpha"
+	"github.com/zitadel/zitadel/pkg/grpc/policy/v2alpha"
+)
+
+func (s *Server) GetLoginPolicy(ctx context.Context, req *policy.GetLoginPolicyRequest) (*policy.GetLoginPolicyResponse, error) {
+	orgID := req.GetOrganisation().GetOrgId()
+	if orgID == "" {
+		orgID = authz.GetCtxData(ctx).OrgID
+	}
+
+	current, err := s.query.LoginPolicyByID(ctx, true, orgID, false)
+	if err != nil {
+		return nil, err
+	}
+	return &policy.GetLoginPolicyResponse{
+		Policy: modelLoginPolicyToPb(current),
+		Details: &object.Details{
+			Sequence:          current.Sequence,
+			ChangeDate:        timestamppb.New(current.ChangeDate),
+			ResourceOwner:     current.OrgID,
+			ResourceOwnerType: IsDefaultToResourceOwnerTypePb(current.IsDefault),
+		},
+	}, nil
+}
+
+func IsDefaultToResourceOwnerTypePb(isDefault bool) object.ResourceOwnerType {
+	if isDefault {
+		return object.ResourceOwnerType_RESOURCE_OWNER_TYPE_INSTANCE
+	}
+	return object.ResourceOwnerType_RESOURCE_OWNER_TYPE_ORG
+}
```

Comment:
- If there are only 2 possible values, why don't we use a boolean in the proto?

Or

- If we want to support more than 2 values, why don't we use a enum in the database?

Reply:
The whole idea was to provide something we can modify in the future, there are only 2 possible values as there are currently only org and instance as resourceowner.
There is only a boolean in the database because policies are the only thing that can have org or instance as resourceowner currently, which is represented by this boolean.

For that, I want something in the API which I can modify in the future but I don't want to change all the logic in the database as it is not necessary at the moment.